README.md

```diff
@@ -180,15 +180,21 @@ Please note for this to work you should create/update user maven settings (typic
      ...
 
 
-### Installing the jars to the local Maven repository ###
-The recommended command is:
+### Installing ALL the jars to the local Maven repository ###
+The recommended command to build for both Scala 2.12 and 2.13 is:
 
     ./gradlewAll publishToMavenLocal
 
 For backwards compatibility, the following also works:
 
     ./gradlewAll install
 
+### Installing specific projects to the local Maven repository ###
+
+    ./gradlew -PskipSigning :streams:publishToMavenLocal
```

Comment:
A bit odd that `skipSigning` is used here, but not in the `./gradleAll` version. Is there a reason?